pages/contact.tsx

```diff
@@ -1,13 +1,16 @@
 import {
   Box, Button, FormHelperText, MenuItem, TextField, Typography,
 } from '@mui/material';
-import React, { useState } from 'react';
+import React, { useEffect, useState } from 'react';
 
+import usePostRequest from '../components/functional/usePostRequest.tsx';
 import IndividualPageHead from '../components/helper/IndividualPageHead.tsx';
 import ImageContainer from '../components/layout/ImageContainer.tsx';
 import SectionContainer from '../components/layout/SectionContainer.tsx';
 import content from '../content/contact.ts';
 
+const API_ENDPOINT = 'https://gv6n06a4h2.execute-api.us-west-2.amazonaws.com/Prod/contact-form';
```

Comment:
@emilyemilyemilyemilyemilyemily is the API endpoint something we should store in `.env` or is it okay to be public?

Reply:
i think its fine to be here - it's a publicly accessible endpoint anyway and having it live in the code makes it easier for devs to test changes

Reply:
Great!